src/main/java/org/zalando/nakadi/controller/StoragesController.java

```diff
@@ -88,4 +88,16 @@ public StoragesController(final SecuritySettings securitySettings, final Storage
         }
         return Responses.create(result.getProblem(), request);
     }
+
+    @RequestMapping(value = "/storages/default/{id}", method = RequestMethod.PUT)
```

Comment:
I think you can add some tests to `StoragesControllerTest`

Reply:
This is covered by StorageControllerAT.java

Reply:
There you cover the basic use case (which is ok for acceptance test)
But you don't cover the error cases (not authenticated; error from storageService), I think unit test for controller is a perfect place for that.